docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -0,0 +1,85 @@
+
+.. _ref-docker-with-datasource:
+
+Using the docker with an S3 bucket as remote datasource.
+========================================================
+
+Introduction
+--------------
+The Lightly Docker can be used with the Lightly Platform to do
+the following workloads in one single run:
+
+- stream the files from your AWS S3 bucket to your local machine
+- embed all images or video frames
+- sample a subset
```

Comment:
sample a subset using Coreset (maybe link to coreset?)

Reply:
We don't have a good description of coreset anywhere, thus I cannot link to it.